query/execute_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Netflix, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package query
+
+import (
+	"testing"
+
+	"context"
+
+	"github.com/Netflix/p2plab"
+	"github.com/stretchr/testify/require"
+)
+
+func GetLabeled() []p2plab.Labeled {
+	return []p2plab.Labeled{
+		NewLabeled("apple", []string{"everyone", "apple", "slowdisk", "region=us-west-2"}),
+		NewLabeled("banana", []string{"everyone", "banana", "region=us-west-2"}),
+		NewLabeled("cherry", []string{"everyone", "cherry", "region=us-east-1"}),
+	}
+}
+
+func TestExecute(t *testing.T) {
+	ctx := context.TODO()
+	ls := GetLabeled()
+
+	labeledSet, error := Execute(ctx, ls, "apple")
```

Comment:
In golang, we usually use `err` as the variable name for errors. `error` is a reserved syntax for the error type, so I'm surprised you were able to use it as a variable name.